.editorconfig

```diff
@@ -0,0 +1,21 @@
+[*.{kt,kts}]
+end_of_line = lf
+ij_kotlin_allow_trailing_comma = false
+ij_kotlin_allow_trailing_comma_on_call_site = false
```

Comment:
Should be `true`

Reply:
Although it's possible that setting this to true was adding the trailing comma after the closure, in which case the workaround was to remove the line and let it default if I remember correctly. But let's try setting `true` first and see how it behaves.

Reply:
updated